snap-geometry-edits/README.md

```diff
@@ -65,6 +65,8 @@ Snapping can be used during interactive edits that move existing vertices using
 
 Geometry guides are enabled by default when snapping is enabled. These allow for snapping to a point coinciding with, parallel to, perpendicular to, or extending an existing geometry.
 
+On supported platforms haptic feedback on `SnapState.snappedToFeature` and `SnapState.snappedToGeometryGuide` is enabled by default when snapping is enabled. Custom haptic feedback can be configured by setting `SnapSettings.isHapticFeedbackEnabled` to false and listening to `GeometryEditor.snapChanged` events to provide specific feedback depending on the `SnapState`.
```

Comment:
```suggestion
On supported platforms, haptic feedback on `SnapState.snappedToFeature` and `SnapState.snappedToGeometryGuide` is enabled by default, when snapping is enabled. Custom haptic feedback can be configured by setting `SnapSettings.isHapticFeedbackEnabled` to false and listening to `GeometryEditor.snapChanged` events to provide specific feedback, depending on the `SnapState`.
```

Reply:
I wonder if we should use the term SharedFlow or Flow at any point here to make it more kotlin specific

Reply:
Here's an example of SharedFlow in doc: https://developers.arcgis.com/kotlin/api-reference/arcgis-maps-kotlin/com.arcgismaps.realtime/-custom-dynamic-entity-data-source/-entity-feed-provider/feed.html

Happy to skip commas, just a suggestion :D